### PR TITLE
update mara_description

### PR DESCRIPTION
**Pull request by YueErro**
Updated according to https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/11.
Tested in 2 PCs.

### CODE REVIEW

**Comment by nzlz** [comment]
Is `mara_contact_publisher` supposed to be updated in this PR too?

**Comment by YueErro** [comment]
Yes for gym-gazebo2.

File: mara_contact_publisher/src/ContactPublisher.cpp [hunk 1]
```diff
@@ -44,27 +44,27 @@ void cb(ConstContactsPtr &_msg)
 
 
   if (_msg->contact_size() > 0){
-    if (_msg->contact(0).collision1() != "ground_plane::link::collision" && _msg->contact(0).collision2() != "ground_plane::link::collision"){
+    // if (_msg->contact(0).collision1() != "ground_plane::link::collision" && _msg->contact(0).collision2() != "ground_plane::link::collision"){
```
Comment (ahcorde) [inline mara_contact_publisher/src/ContactPublisher.cpp]: Remove this comment

File: mara_contact_publisher/src/ContactPublisher.cpp [hunk 2]
```diff
@@ -44,27 +44,27 @@ void cb(ConstContactsPtr &_msg)
...
+      normals.x =  _msg->contact(0).normal(j).x();
+      normals.y =  _msg->contact(0).normal(j).y();
+      normals.z =  _msg->contact(0).normal(j).z();
+      contact.contact_normals.push_back(normals);
 
-        contact.depths.push_back(static_cast<float>(_msg->contact(0).depth(0)));
-      }
+      contact.depths.push_back(static_cast<float>(_msg->contact(0).depth(0)));
     }
+    // }
```
Comment (ahcorde) [inline mara_contact_publisher/src/ContactPublisher.cpp]: remove this line

**Comment by nzlz** [comment]
Few issues when running (gazebo 9.0.0):
`ros2 launch mara_gazebo mara.launch.py --urdf mara_robot_gripper_140` : gripper is out of scale.
`ros2 launch mara_gazebo mara.launch.py --urdf mara_robot_gripper_85` : gripper is malformed.
`ros2 launch mara_gazebo mara.launch.py --urdf mara_robot_gripper_hande` : gripper has bad orientation, looking towards -x axis.

**Comment by YueErro** [comment]
@nzlz 
>ros2 launch mara_gazebo mara.launch.py --urdf mara_robot_gripper_140 : gripper is out of scale.

Can you tell me which component is out of scale?
If you check the generated [urdf](https://github.com/AcutronicRobotics/MARA/blob/mara_description/mara_description/urdf/mara_robot_gripper_140.urdf) all the daes are scaled. Should be fine.

**Comment by nzlz** [comment]
@YueErro  Removed my comment, I had another branch.

Can you test that `ros2 launch mara_gazebo mara.launch.py --urdf mara_robot_gripper_85` is working for you? For me it is not, but not sure if its my systems problem. Same for other grippers using `rework` branch in robotiq.

**Comment by YueErro** [comment]
Yes, i have already tested before making the PR.

**Comment by YueErro** [comment]
I am with Gazebo 9.9.0.

**Comment by nzlz** [comment]
Update both libgazebo9 and libgazebo9-dev to get latest 9.9.0 solved my issues, working fine now. If gazebo 9.9 is a must now, we should take care of this for both instructions and dockerfiles, or maybe I just had a broken installation.

**Pull request merged**